pkg/controller/accountclaim/accountclaim_controller.go

```diff
@@ -324,6 +324,9 @@ func (r *ReconcileAccountClaim) Reconcile(request reconcile.Request) (reconcile.
 	if accountClaim.Spec.AccountOU == "" || accountClaim.Spec.AccountOU == "ROOT" {
 		err = MoveAccountToOU(r, reqLogger, accountClaim, unclaimedAccount)
 		if err != nil {
+			if err == awsv1alpha1.ErrAccMoveRaceCondition {
+				return reconcile.Result{Requeue: true}, nil
```

Comment:
This would be a good spot for a comment like the one [below](https://github.com/openshift/aws-account-operator/pull/411/files#diff-90b216857326e9f99796eb1d9cefbc91R130).

Reply:
++